sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveQl.scala

```diff
@@ -454,7 +455,8 @@ https://cwiki.apache.org/confluence/display/Hive/Enhanced+Aggregation%2C+Cube%2C
            Token("TOK_TABNAME", tableNameParts) ::
            ifExists) =>
       val tableName = tableNameParts.map { case Token(p, Nil) => p }.mkString(".")
-      DropTable(tableName, ifExists.nonEmpty)
+      // not a temp table, temporary false
+      DropTable(tableName, ifExists.nonEmpty, false)
```

Comment:
I think we can add an `isTemporary` field to Hive `DropTable`, so that we don't bother introducing the newly introduced logical `DropTable` plan here.